app/controllers/api/automate_workspaces_controller.rb

```diff
@@ -1,9 +1,9 @@
 module Api
   class AutomateWorkspacesController < BaseController
     def show
-      obj = AutomateWorkspace.find_by(:guid => @req.c_id)
+      obj = AutomateWorkspace.find_by(:guid => @req.collection_id)
```

Comment:
THANK YOU... c_id and those shortcuts have been killing me forever :)

Reply:
Who are these people that like to have shortcuts 😉